1-Initial-Setup/1.4-Secure-boot-settings.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+@test "1.4.1 Ensure permissions on bootloader config are not overridden (Automated)" {
+    run bash -c 'grep -E '\''^\s*chmod\s+[0-7][0-7][0-7]\s+\$\{grub_cfg\}\.new'\'' -A 1 -B1 /usr/sbin/grub-mkconfig'
+    [ "$status" -eq 0 ]
+    [[ "$output" == 'if [ "x${grub_cfg}" != "x" ]; then'* ]]
+    [[ "$output" == *'  chmod 400 ${grub_cfg}.new || true'* ]]
+    [[ "$output" == *"fi" ]]
```

Comment:
Here you could query exactly the lines with the variable $lines.

```suggestion
    [[ "${lines[0]} == 'if [ "x${grub_cfg}" != "x" ]; then' ]]
    [[ "${lines[1]} == '  chmod 400 ${grub_cfg}.new || true' ]]
    [[ "${lines[2]} == *"fi" ]]
```

Reply:
good catch!

Reply:
Well, you forgot the trailing quotes...